cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/EnvironmentDTO.java

```diff
@@ -1850,6 +1850,11 @@ public enum SelfBilling
             PER_PLAN
         }
 
+        public enum ResellerMarketplace
+        {
+            ENABLED, EXCLUSIVE, DISABLED
+        }
```

Comment:
Non ho ben capito questo

L'ho inteso come:
- ENABLED, parent child attivo con un generico marketplace di piattaforma
- EXCLUSIVE, parent child attivo senza un generico marketplace di piattaforma
- DISABLED, parent child spento

Reply:
parliamo di [questa feature](https://docs.cloudesire.com/docs/channel.html#automated-distributors-and-resellers-marketplace-creation)

scrivo del giavadocco